app/src/org/commcare/activities/CommCareSetupActivity.java

```diff
@@ -637,36 +665,6 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return true;
     }
 
-    /**
-     * Return to or launch dispatch activity.
-     *
-     * @param failed did installation occur successfully?
-     */
-    private void done(boolean failed) {

```

Comment:
this was only getting called in 1 place, by `reportSuccess`, so it seemed odd to have it separated out since it was just a continuation of what's going on there.